deepblink/networks/fcn.py

```diff
@@ -11,27 +11,31 @@
 
 
 def fcn(
-    dropout: float = 0.2, cell_size: int = 4, filters: int = 6, n_convs: int = 3,
+    dropout: float = 0.2,
+    cell_size: int = 4,
+    filters: int = 6,
+    n_convs: int = 3,
+    n_extra_down: int = 0,
 ) -> tf.keras.models.Model:
     """Simplest FCN architecture without skips or dropout.
 
     Arguments:
-        dropout: Percentage of dropout only for resnet architecture.
+        dropout: Percentage of dropout after each conv layer.
         cell_size: Size of one cell in the prediction matrix.
         filters: Log2 number of filters in the first convolution layers.
         n_convs: Number of convolution layers in each convolution block.
+        n_extra_down: extra downsampling followed by same number of up sampling.
```

Comment:
For a later commit probably change `n_extra_down`. Slightly strange name but don't have a better one in mind just yet.